helm-charts/konk-operator/templates/deployment.yaml

```diff
@@ -35,6 +35,8 @@ spec:
           env:
             - name: AUTH_URL
               value: {{ .Values.authURL | default "" }}
+            - name: CERT_MANAGER_NAMESPACE
+              value: {{ (index .Values "cert-manager").namespace }}
```

Comment:
wouldn't it have been easier to remove the `-` ? I don't see any proprietary code defining "cert-manager" today.

Reply:
When installed as a subchart, cert-manager's values are under `cert-manager`.